css/styles.css

```diff
@@ -0,0 +1,71 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
- [ ] [OPTIONAL] kindly make your design match the Figma design your page should have a max width of 100% so that the hamburger menu should be directly visible without scrolling your page . :grinning: 

![image](https://user-images.githubusercontent.com/95297251/185523400-290d752d-48ab-403e-9117-45c96a51498b.png)

Reply:
@MaganaAsiati Thanks.
I copied all the images from Figma template. 
I also use the google.fonts.